packages/E2ETest/metro.config.js

```diff
@@ -37,19 +37,10 @@ module.exports = {
     // Since there are multiple copies of react-native, we need to ensure that metro only sees one of them
     // This should go away after RN 0.60 when haste is removed
     blacklistRE: blacklist([
-      new RegExp(
-        `.*E2ETest/msbuild.*`.replace(/[/\\]/g, '\\/')
-      ), // Avoid error EBUSY: resource busy or locked, open 'D:\a\1\s\packages\E2ETest\msbuild.ProjectImports.zip' in pipeline
+      new RegExp('.*E2ETest/msbuild.*'.replace(/[/\\]/g, '\\/')), // Avoid error EBUSY: resource busy or locked, open 'D:\a\1\s\packages\E2ETest\msbuild.ProjectImports.zip' in pipeline
       new RegExp(`${path.resolve(rnPath)}.*`.replace(/[/\\]/g, '/')),
       new RegExp(
-        `${path
-          .resolve(rnwPath, 'RNTesterCopy')
-          .replace(/[/\\]/g, '/')}.*`,
-      ),
-      new RegExp(
-        `${path
-          .resolve(rnwPath, 'IntegrationTestsCopy')
-          .replace(/[/\\]/g, '/')}.*`,
+        `${path.resolve(rnwPath, 'ReactCopies').replace(/[/\\]/g, '/')}.*`
```

Comment:
>'/' [](start = 66, length = 3)

(nit) consider having some method "NormilizePathSeparator" and have "/" be const. Similar as it's done in C# Path.* Apis. That way it will be easier to deal with paths

Reply:
I'm not sure there's a nice way to share logic between metro config files in different packages. Ironically the reason they need to do this replacement is because Node will normalize to Windows style slashes, and seemingly Metro expects Unix style. 

We should be able to avoid the sketchy blacklisting stuff soon though, since Facebook has committed to publishing these.